using-mobile-extensions/adobe-analytics/analytics-event-reference.md

```diff
@@ -2,13 +2,13 @@
 
 ## Internal Events for Adobe Analytics
 
-In addition to the Analytics hits that are sent as a result of the track call, the Adobe Experience Cloud Platform SDKs also tracks some internal events that are based on the communication with other modules.
+In addition to the Analytics hits that are sent as a result of the track call, the Adobe Experience Platform SDKs also tracks some internal events that are based on the communication with other extensions.
```

Comment:
In the first sentence, please change "tracks" to "track" so that it reads "...the Adobe Experience Platform SDKs also track some..."